docs/sourcemaps.md

```diff
@@ -0,0 +1,86 @@
+---
+id: sourcemaps
+title: Source Maps
+---
+
+Source maps help map a minified file back to an un-packaged state. This is common when investigating issues from release builds.
+
+This helps if you have a stacktrace like
+
+```text
+TypeError: Cannot read property 'data' of undefined
+  at anonymous(app:///index.android.bundle:1:4277021)
+  at call(native)
+  at p(app:///index.android.bundle:1:227785)
+```
+
+The power of source maps allow tracing that line and column number back to the original source stacktrace.
+
+```text
+TypeError: Cannot read property 'data' of undefined
+  at anonymous(src/modules/notifications/Permission.js:15:requestNotificationPermission)
+  at call(native)
+  at p(node_modules/regenerator-runtime/runtime.js:64:Generator)
+```
+
+This allows to triage release issues given a more accurate stacktrace.
+
+Follow the instructions below to get started with source maps:
+
+1. [Enable source maps on Android](sourcemaps.md#enable-source-maps-on-android)
+2. [Enable source maps on iOS](sourcemaps.md#enable-source-maps-on-ios)
+3. [Manual Symbolication](sourcemaps.md#manual-symbolication)
```

Comment:
```suggestion
Follow the instructions below to get started with source maps.
```

Since we have ToC on the right side, I think we don't need to list the sections in content.